CPAProxy/CPAProxyManager.m

```diff
@@ -164,7 +164,9 @@ - (void)tryConnectingControlPortAfterDelay:(NSTimeInterval)delay
 
 - (void)connectSocket
 {
-    self.controlPortStatus = CPAControlPortStatusConnecting;
+    if (self.controlPortStatus != CPAControlPortStatusConnecting) {
```

Comment:
Why setting controlPortStatus to CPAControlPortStatusConnecting removed from here?